examples/comparison/gorm4/autogenerated_gorm4.go

```diff
@@ -153,25 +159,25 @@ func (qs UserQuerySet) GetUpdater() UserUpdater {
 
 // IDEq is an autogenerated method
 // nolint: dupl
-func (qs UserQuerySet) IDEq(ID uint) UserQuerySet {
+func (qs UserQuerySet) IDEq(ID uint64) UserQuerySet {
```

Comment:
why did you change `uint` to `uint64` everywhere? it's the incompatible change

Reply:
I didn't did anything for above mentioned change. I am not able to figure out the root cause of it. Can we start the discussion in order to resolve this issue?